client/.env.example

```diff
@@ -1 +1 @@
-REACT_APP_API_URL=http://localhost:5000/api
\ No newline at end of file
+REACT_APP_API_URL=http://localhost:3000/api
```

Comment:
why did you changed this? keep in mind that the server is listening to requests from port 5000 so this might cause an issue (this is not the client url - this means that the client can ,ake network request to the server by using this url)

Reply:
The server wasn't working on port 5000 on my computer because of a potential conflict or configuration issue. I also changed the client URL to ensure that the client can correctly communicate with the server. Now, the client can make network requests to the server using the updated URL.

Reply:
@ReutMesi ok, so in general if you need to change configuration due to local env issues, these changes should not be committed since it can affect others (and potentially 'production'). so just rollback this file the the original setup and if you need to change it locally just remember to revert the changes.